### ISSUE TITLE
Group Custom Question Bank

### ISSUE THREAD

**Issue by orenmendelow** [yosefserkez/mailpals#23]
It would be cool if- especially for corporate settings, but even for friends/family groups, there could be a custom question bank. We could even give an editable prompt to users to pass to chatGPT- this would save any AI API calls on Mailpals end and allow users to curate their own questions.

For example, if I were to make a mailpals group with coworkers, I'd want questions that are specific to our work environment/ culture.

This can be done by toggling an option in group settings under # of questions option, which would reveal a textbox. Questions can be parsed just by newline (to account for questions that are 2-in-1's).

Bonus: an additional toggle to opt between "only pick from this question bank" or "use a combination of these questions and mailpals-suggested ones"

<img width="899" alt="Screenshot 2024-10-29 at 6 24 07 PM" src="https://github.com/user-attachments/assets/8f91a0a4-442a-4af6-b178-b966f2836434">

**Comment by yosefserkez** [comment]
Good idea. A part of this would also be to allow members to suggest questions for the question bank / next issue

**Comment by orenmendelow** [comment]
Totally. Two questions-
1. Are you saying that you'd have the site ingest these custom questions and build a database for use in similar groups? No clue how that would work but sounds cool.
2. How to keep track of which questions were already asked to eliminate repeats? Do we care about repeats?
